content/en/users/compute/orchestration/dodas/_index.md

```diff
@@ -1,13 +1,36 @@
 ---
 title: Dynamic On-Demand Analysis Software
 linkTitle: Dynamic On-Demand Analysis Software
-draft: true
 type: docs
 weight: 40
 description: >
-  Subtitle here
+  Instantiate on-demand customizable infrastructures for data analysis
 ---
 
 ## What is it?
 
-TBD
+DODAS enables the execution of user analysis code both in batch mode and
```

Comment:
Do not use acronyms without introducing them first. On the first occurrence in the text*, use the full name (in this case "Dynamic On-Demand Analysis Software") followed by the acronym in between parenthesis. The first full use of the service name should also be a link to the actual service**.

* in titles always use the full name
** never put links in titles, though